onedal/svm/svm.py

```diff
@@ -365,12 +191,8 @@ def __init__(
             cache_size=cache_size,
             max_iter=max_iter,
             tau=tau,
-            class_weight=None,
-            decision_function_shape=None,
-            break_ties=False,
             algorithm=algorithm,
```

Comment:
Maybe I do not understand something, but why aren't some of the parameters passed to BaseSVM constructor?
For example, 'tol', 'schrinking', 'gamma', etc?

Reply:
I mean, why some parameters like (cache_size) got propagated to BaseSVM while others do not?
Looks like an inconsistency to me.